spec/lib/content_exporter_spec.rb

```diff
@@ -0,0 +1,199 @@
+locale_results_link_fixture = {
+  group_one: {
+    subgroup_one: {
+      title: "I am the title for Group one subgroup one",
+      items: [
+        {
+          id: "0001",
+          support_and_advice: false,
+          text: "This is a row that only has text, it will appaer like a paragraph",
```

Comment:
Here and elsewhere in the document `appear` is misspelt as `appaer`.

Reply:
:man_facepalming: thanks for catching that!